.dockerignore

```diff
@@ -0,0 +1,18 @@
+# Documentation, might leak version number
```

Comment:
I think this exclusions are still useful, but they rather belong to the "unnecessary files" section. 😄